doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,144 @@
 (network-ovn-setup)=
 # How to set up OVN with LXD
 
+See the following sections for how to set up a basic OVN network, either as a standalone network or to host a small LXD cluster.
+
+## Set up a standalone OVN network
+
+Complete the following steps to create a standalone OVN network that is connected to a managed LXD parent bridge network (for example, `lxdbr0`) for outbound connectivity.
+
+1. Install the OVN tools on the local server:
+
+        sudo apt install ovn-host ovn-central
+
+1. Configure the OVN integration bridge:
+
+        sudo ovs-vsctl set open_vswitch . \
+          external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+          external_ids:ovn-encap-type=geneve \
+          external_ids:ovn-encap-ip=127.0.0.1
+
+1. Create an OVN network:
+
+        lxc network set <parent_network> ipv4.dhcp.ranges=<IP_range> ipv4.ovn.ranges=<IP_range>
+        lxc network create ovntest --type=ovn network=<parent_network>
+
+1. Create an instance that uses the `ovntest` network:
+
+        lxc init ubuntu:22.04 c1
+        lxc config device override c1 eth0 network=ovntest
+        lxc start c1
+
+1. Run `lxc list` to show the instance information:
+
+   ```
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   | NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   | c1   | RUNNING | 192.0.2.2 (eth0)    | 2001:db8:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   ```
+
+## Set up a LXD cluster on OVN
+
 ```{youtube} https://www.youtube.com/watch?v=1M__Rm9iZb8
 ```
 
-This will create a standalone OVN network that is connected to the parent network lxdbr0 for outbound connectivity.
+Complete the following steps to set up a LXD cluster that uses an OVN network.
+The instructions are for a cluster of four servers, where three of the servers run the distributed database for OVN and the OVN controller, and the fourth server runs only the OVN controller.
```

Comment:
Much better thanks. Perhaps we should say why we are choosing this configuration for the tutorial. The reason is (I suppose) to show that both LXD and OVN database role servers need to existing in a cluster of odd numbered members (at least 3, then 5, 7 etc, to avoid split brain scenarios). And then the fourth server to demonstrate how normal non-DB servers are configured.

3 is the minimum size for this setup. Perhaps 5 would be better for a demo though as it would allow us to show that the LXD cluster size doesn't have to match the OVN DB cluster size (i.e 5 LXD cluster members, made up of 3 OVN DB members and 2 OVN controller only members)? cc @stgraber

Reply:
Could it just be up to the user? Thus 3 OVN DB members, and then a random number of OVN controller-only members?

Reply:
Yes indeed. But we can show that the ovn db members don't have to match the lxd cluster members